src/components/NavBar.js

```diff
@@ -32,8 +45,8 @@ const NavBar = props => {
           <Route exact path="/adoptions/new" />
         </Switch>
       </div>
-    </>
-  )
-}
+    </Fragment>
+  );
+};
 
-export default NavBar
+export default NavBar;
```

Comment:
As of ES6 semicolons are no longer needed in these cases! This obviously will not break your code, but it is extraneous and as the potential to cause problems.